wasm-wrappers/src/lib.rs

```diff
@@ -718,10 +736,10 @@ pub fn encode_output_issue_nft(
     ticker: &str,
     description: &str,
     media_hash: &[u8],
-    creator: Option<String>,
-    media_uri: Option<Vec<u8>>,
-    icon_uri: Option<Vec<u8>>,
-    additional_metadata_uri: Option<Vec<u8>>,
+    creator: Option<Vec<u8>>,
+    media_uri: Option<String>,
+    icon_uri: Option<String>,
+    additional_metadata_uri: Option<String>,
```

Comment:
Why aren't these `Option<&str>` and `Option<&[u8]>`?

Reply:
I think the wasm bindings generator doesn't know how to handle it.